### PR TITLE
Feature/pb 29 recent posts pagination

### PR DESCRIPTION
**Pull request by GabrielCFurlan**
FrontEnd with buttons and data fetched
image

### CODE REVIEW

File: src/components/RecentPosts/RecentPosts.style.js [hunk 1]
```diff
@@ -10,8 +10,37 @@ export const Title = styled.div`
...
+`
+
+export const Button = styled.button`
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  font-size: ${({ theme }) => theme.fontSizes.medium};
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+  color: ${({ theme }) => theme.colors.primary};
+  margin-left: 15px;
```
Comment (Jejugo) [inline src/components/RecentPosts/RecentPosts.style.js]: use standard style spacing set on `theme.js` and you can use only one CSS property `margin: 0px 15px`

File: src/components/RecentPosts/index.jsx [hunk 2]
```diff
@@ -1,55 +1,66 @@
...
-    category: "Food",
-    title: "Navigating First Impressions: Introduce Yourself",
-    link: "/",
-    description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-  },
-]
-export default function MostPopular() {
+
+// função pra reuquisição do banco
```
Comment (Jejugo) [inline src/components/RecentPosts/index.jsx]: remove unnecessary comments, and please do not make any typos next time

**Review by Jejugo — APPROVED** [review]

**Review by Cacatuaa — APPROVED** [review]

File: src/components/RecentPosts/RecentPosts.style.js (hunk 1 above)
Comment (GabrielCFurlan) [reply src/components/RecentPosts/RecentPosts.style.js]: Adjusted!

File: src/components/RecentPosts/index.jsx (hunk 2 above)
Comment (GabrielCFurlan) [reply src/components/RecentPosts/index.jsx]: Adjusted!

File: src/components/RecentPosts/RecentPosts.style.js [hunk 3]
```diff
@@ -10,8 +10,37 @@ export const Title = styled.div`
...
+`
+
+export const Button = styled.button`
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  font-size: ${({ theme }) => theme.fontSizes.medium};
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+  color: ${({ theme }) => theme.colors.primary};
+  margin: ${({ theme }) => theme.spacings.medium}
```
Comment (Jejugo) [inline src/components/RecentPosts/RecentPosts.style.js]:
```
${({ theme }) => `0px ${theme.spacings.medium}`}
```

**Pull request merged**